From: Mattéo Rossillol‑‑Laruelle <beatussum@protonmail.com>
Subject: override `config/common`
Signed-off-by: Mattéo Rossillol‑‑Laruelle <beatussum@protonmail.com>
Description:
This patch overrides the `config/common` file created by `./build.sh`

--- a/config/common
+++ b/onfig/common
@@ -40,13 +40,13 @@
 LB_DEBCONF_FRONTEND="noninteractive"

 # Set debconf(1) priority to use
-LB_DEBCONF_PRIORITY="critical"
+LB_DEBCONF_PRIORITY="high"

 # Set initramfs hook
 LB_INITRAMFS="live-boot"

 # Set initramfs compression
-LB_INITRAMFS_COMPRESSION="gzip"
+LB_INITRAMFS_COMPRESSION="lzma"

 # Set init system
 LB_INITSYSTEM="systemd"
@@ -58,7 +58,7 @@
 LB_SYSTEM="live"

 # Set base name of the image
-LB_IMAGE_NAME="live-image"
+LB_IMAGE_NAME="bsum-kali"

 # Set options to use with apt
 APT_OPTIONS="--yes"
@@ -72,8 +72,8 @@
 # Set script to use with debootstrap
 DEBOOTSTRAP_SCRIPT=""

-# Set options to use with gzip
-GZIP_OPTIONS="-6 --rsyncable"
+# Set options to use with lzma
+LZMA_OPTIONS="-5 -T $(( $(nproc) / 2 )) --verbose"

 # Enable UTC timestamps
 LB_UTC_TIME="false"
@@ -90,7 +90,7 @@

 # Enable color
 # If set here, overrides the command line option
-#_COLOR="auto"
+_COLOR="true"

 # Enable force
 # If set here, overrides the command line option
@@ -102,4 +102,4 @@

 # Enable verbose
 # If set here, overrides the command line option
-#_VERBOSE="false"
+_VERBOSE="true"
